### DISCUSSION TITLE
Unable to login using the old credentials after updating the Budibase Self-hosted Version

### DISCUSSION THREAD

**Discussion by starsang1995** [Budibase/budibase#10107] [Help]
Hi I am not able to login now using my old credentials after updating my instance of Budibase to the latest version in app. Also the reset password workflow is not working either, does anyone know how to fix this? Many thanks ahead!

**Comment by mike12345567** [comment]
Hi @apsonandtesla - how are you hosting your Budibase installation?

Sometimes the in-app update doesn't work as expected, we need to remove it as its not really a safe way of updating, you should instead us the Budibase CLI to update: https://docs.budibase.com/docs/updating-budibase
